src/main/java/net/neoforged/neoforge/network/registration/NetworkRegistry.java

```diff
@@ -622,7 +608,7 @@ public static boolean hasChannel(Connection connection, @Nullable ConnectionProt
      * @return The filtered packets.
      */
     public static <T extends PacketListener> List<Packet<?>> filterGameBundlePackets(ChannelHandlerContext context, Iterable<Packet<? super T>> packets) {
-        final NetworkPayloadSetup payloadSetup = context.channel().attr(ATTRIBUTE_PAYLOAD_SETUP).get();
+        NetworkPayloadSetup payloadSetup = context.channel().attr(ChannelAttributes.PAYLOAD_SETUP).get();
```

Comment:
Why not use the `ChannelAttributes#getPayloadSetup` method?

Reply:
Because those helpers work on `Connection` (to avoid unwrapping the `Channel` at most call sites), but this call site is unique in that we don't have a `Channel`, we have a `ChannelHandlerContext`.

And while true they both have a method with signature `public Channel channel()`, there isn't a common interface to retrieve it by.